repairnator/repairnator-realtime/src/main/java/fr/inria/spirals/repairnator/realtime/InspectProcessDuration.java

```diff
@@ -17,18 +17,19 @@
     private EndProcessNotifier endProcessNotifier;
     private InspectBuilds inspectBuilds;
     private InspectJobs inspectJobs;
-    private DockerPipelineRunner pipelineRunner;
+    private DockerPipelineRunner DockerPipelineRunner;
+    private ActiveMQPipelineRunner ActiveMQPipelineRunner;
 
-    public InspectProcessDuration(InspectBuilds inspectBuilds, InspectJobs inspectJobs, DockerPipelineRunner pipelineRunner, EndProcessNotifier endProcessNotifier) {
-        this(inspectBuilds, inspectJobs, pipelineRunner);
+    public InspectProcessDuration(InspectBuilds inspectBuilds, InspectJobs inspectJobs, DockerPipelineRunner DockerPipelineRunner, EndProcessNotifier endProcessNotifier) {
```

Comment:
parameter names never start with upper case in Java

Reply:
Ah sorry, this should have been fixed previously but after I copy pasted from my previous PR , this appears again :) .

Reply:
It is fixed :)